src/main/java/fi/evident/elasticsearch/voikko/analysis/FinnishTokenizerFactory.java

```diff
@@ -32,7 +32,7 @@ public FinnishTokenizerFactory(IndexSettings indexSettings,
                                    @SuppressWarnings("unused") Environment environment,
                                    @SuppressWarnings("unused") String name, // TODO do we need name somewhere?
```

Comment:
You can remove `@SuppressWarnings("unused")` and the TODO now that this is actually used.